CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## CHANGE LOG
 
+### v1.0.0 Update to next 12
```

Comment:
Maybe title this as "Remediation Project (security) update"? Not sure security is needed.